addons/stock_dropshipping/tests/test_lifo_price.py

```diff
@@ -86,7 +86,7 @@ def test_lifoprice(self):
         with out_form.move_ids_without_package.new() as move:
             move.product_id = product_lifo_icecream
             move.quantity_done = 20.0
-            move.date_expected = fields.Date.context_today(self.env['stock.move.line'])
+            move.date_expected = fields.Datetime.now(self.env['stock.move.line'])
```

Comment:
Weird, should crash. should perhaps preserve logic with context_today and the TZ in env.

Reply:
Actually I think this is correct, date_expected is actually a Datetime and now accepts a tzinfo argument, TBD

Reply:
datetime.now() does accept a tzinfo arg, but **D**atetime.now() ignores its arguments, right? So it will not crash, but the argument is useless, since we only deal with UTC datetimes for field values.